en/docs/setup/wso2is_script.sql

```diff
@@ -0,0 +1,83 @@
+ALTER TABLE `apimgt`.`IDN_OAUTH2_ACCESS_TOKEN` CHANGE COLUMN `TIME_CREATED` `TIME_CREATED` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP , CHANGE COLUMN `REFRESH_TOKEN_TIME_CREATED` `REFRESH_TOKEN_TIME_CREATED` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ;

```

Comment:
Shall we remove the DB name (`apimgt`) part from the query? The DB name is variable.